packages/shared/src/components/profile/Header.tsx

```diff
@@ -30,30 +34,37 @@ export function Header({
     cid: ReferralCampaignKey.ShareProfile,
     logObject: () => ({ event_name: 'share profile', target_id: user.id }),
   });
+  const isMobile = useViewSize(ViewSize.MobileL);
+  const notificationsNavBar = useFeature(feature.notificationsNavBar);
```

Comment:
We probably want to limit enrollment to only mobile users, maybe use `useConditionalFeature` here to be safe :)

Reply:
Used useConditionalFeature